src/components/Bars/TooltipBar.js

```diff
@@ -0,0 +1,54 @@
+import React, { useReact, useEfect, Fragment, useEffect, useState } from 'react';
+import styled from 'styled-components'
+
+const Text = styled.span`
+    color: #000;
+`;
+
+const ButtonStylized = styled.button`
+
+    width: 129px;
+    height: 41px;
+    border-radius: 6px;
+    background-color: ${props => props.color};
+    margin: 0 15px;
+    border: none;
+
+    ${Text} { 
+        color: white;
+    }
+`;
+
+const BoxOfShadows = styled.div`
+    width: 1590px;
+    height: 166px;
+    box-shadow: 0px 10px 20px 0 rgba(229, 229, 230, 0.75);
+    background-color: #ffffff;
+
+    h2 {
+        margin-left: 15px;
+    }
+
+    Fragment {
+        display: flex;  
+        align-items: center;
+    }
```

Comment:
Please don't use react elements for referencing in the CSS, as, and especially the `Fragment`, it doesn't render as it's named. Btw, the Fragment doesn't get rendered at all, it leaves only in the virtual DOM.